docs/decisions/0023-ai-pdf-transcription.md

```
@@ -0,0 +1,271 @@
+data ImageUrl = ImageUrl
+  { url :: Text
+  -- ^ Data URL (e.g., "data:application/pdf;base64,...")
+  }
+  deriving (Show, Eq, Generic)
+```
+
+### 3. Architecture: Piggyback on OpenRouter
+
+The integration follows the piggyback pattern used by `Pdf.ExtractText.Internal`:
+
+```
```

markdownlint-cli2 (0.20.0)

[warning] Fenced code blocks should have a language specified

(MD040, fenced-code-language)